test/extended/gssapi.sh

```diff
@@ -0,0 +1,290 @@
+#!/bin/bash
+#
+# TODO
+
+# set -x
+
+set -e
```

Comment:
The following statements are equivalent
```sh
set -e
set -o errexit
```

Since you've got the latter on line 9, I don't think you need this one